### PR TITLE
Fix not able to input underscore.

### PR DESCRIPTION
**Pull request by franciscomorais**
## Proposed Changes
Change the underscore unicode in keyMap.

## Testing
- tapOn: input
-  inputText: foo_bar

## Issues Fixed
https://github.com/mobile-dev-inc/maestro/issues/365

### CODE REVIEW

File: maestro-ios/src/main/java/ios/idb/TextInputUtil.kt [hunk 1]
```diff
@@ -195,7 +195,7 @@ internal object TextInputUtil {
         '*' to keyPressShiftedToEvents(37),
         '(' to keyPressShiftedToEvents(38),
         ')' to keyPressShiftedToEvents(39),
-        '"' to keyPressShiftedToEvents(45),
+        '_' to keyPressShiftedToEvents(45),
```
Comment (dmitry-zaitsev) [inline maestro-ios/src/main/java/ios/idb/TextInputUtil.kt:198]: isn't underscore code 95?

Comment (franciscomorais) [reply maestro-ios/src/main/java/ios/idb/TextInputUtil.kt:198]: Sorry @dmitry-zaitsev .. I can't test. The truth is that I was doing a flow and I noticed that the underscore is replaced by "". I found this issue and decided to PR it.

If you want I can change it to 95 but I can't really test it.

Comment (franciscomorais) [reply maestro-ios/src/main/java/ios/idb/TextInputUtil.kt]: @dmitry-zaitsev Okay, I ended up change it. Please, take another look. Thanks.

Comment (dmitry-zaitsev) [reply maestro-ios/src/main/java/ios/idb/TextInputUtil.kt]: I checked your changes and your original version was correct. It should be 45

Sorry for the confusion

**Comment by dmitry-zaitsev** [comment]
Could you please exclude version number changes from this PR?

**Review by jrelorcasa — APPROVED** [review]

**Comment by dmitry-zaitsev** [comment]
Thanks for pointing out the solution! Since you did not update PR I just pushed the fix myself

**Comment by tom-daggett-wd** [comment]
Note that what got pushed in this PR is this change:
`'_' to keyPressShiftedToEvents(95),`
but from the comments, I  thought people agreed that the change *should* be:
`'_' to keyPressShiftedToEvents(45),`

**Comment by Leland-Takamine** [comment]
@tom-daggett-wd This is the PR that ended getting merged: https://github.com/mobile-dev-inc/maestro/pull/411

**Pull request closed without merging**